src/assets/styles/_margins.scss

```diff
@@ -7,6 +7,6 @@
 
 .outer-margin {
   margin: auto;
-  padding-left: calc(var(--padding-large) + 2vw);
-  padding-right: calc(var(--padding-large) + 2vw);
+  margin-left: calc(var(--margin-large) + 2vw);
```

Comment:
![image](https://github.com/PonderTeam/ponder-app/assets/39470736/8e637307-13e3-4efc-ad89-e689665343f0)
left doesn't seem like its aligned

Reply:
![image](https://github.com/PonderTeam/ponder-app/assets/39470736/ec359eee-f4f4-43f1-8090-ec9c88adfc1f)
this is no padding-left when toggled off in the dev tool

![image](https://github.com/PonderTeam/ponder-app/assets/39470736/541f51e8-5664-4b0c-926b-ded32842cd69)
this is with padding-left when toggled on
it looks like the border is stuck to the left card instead of the side of the screen?

Reply:
It could also possibly be that I'm on a 24in monitor?

Reply:
Sam was not on the proper branch when he made these requests. We figured it out together 😄